src/styles/app.styl

```diff
@@ -470,6 +470,7 @@ footer
     display flex
     justify-content center
     align-items stretch
+    margin-top 27px
```

Comment:
Ok, I do the modification. 27 is the value that makes the design match de request from Zoe.